tizen/src/platform_tizen.cpp

```diff
@@ -220,9 +226,103 @@ std::string stringFromFile(const char* _path) {
     return out;
 }
 
-// No system fonts fallback implementation (yet!)
+void initPlatformFontSetup() {
```

Comment:
Hmm.. we can probably do this in `FontContext` constructor! Which means for every scene load/update this will be called.